### ISSUE TITLE
App does not see existing profiles/settings from other instance of app on dual-booted OS

### ISSUE THREAD

**Issue by Pyroglyph** [modrinth/code#3006]
### Please confirm the following.

- [x] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [x] I have tried resolving the issue using the [support portal](https://support.modrinth.com)
- [x] I have ensured my Modrinth App installation is up to date

### What version of the Modrinth App are you using?

0.8.9

### Describe the bug

The Modrinth app is unable to "see" profiles I have created on Windows when I install it on Linux and point the "app directory" setting to the same directory.

### Steps to reproduce

1. Install the Modrinth app
2. Make a note of the app directory setting (or set it to something you can remember)
3. Create an instance
4. Boot into another OS
5. Install the Modrinth app
6. Point the "app directory" setting to the same one you noted down
7. Restart the launcher and see that your instance is still missing

### Expected behavior

Pointing the Modrinth app directory to a directory used by another instance of the app on another OS should mean that all of my instances, login data, settings, etc, should be present on both OS's.
In other words, changes I make in the app on one OS should be seen when I launch the app on the other OS, given they are pointed to the same app directory.

**Comment by manen** [comment]
i'm having a similar issue where after moving the app directory to a secondary drive (and of course setting the app directory to the new path in the settings) all the data i had in modrinth like minecraft account and profiles disappear, while the ModrinthApp folder is fully intact, and where it's supposed to be.
also a dual-booted pc, problem appearing on windows
